spec/fixtures/exports/case_logs.xml

```diff
@@ -158,6 +158,23 @@
     <previous_postcode_known>1</previous_postcode_known>
     <previous_la_known/>
     <is_previous_la_inferred>false</is_previous_la_inferred>
+    <age1_known/>
+    <age2_known/>
+    <age3_known/>
+    <age4_known/>
+    <age5_known/>
+    <age6_known/>
+    <age7_known/>
+    <age8_known/>
+    <ethnic_group/>
```

Comment:
Should this be `ethnic` rather than `ethnic_group`? That's the field that stores the final information ADD want right?

Reply:
this have been split across two fields now but it could be omitted if necessary

Reply:
now omitted

Reply:
Right. Those two field will need to be recombined into the original field then (and that exported) so the data sent to CDS remains the same.

Reply:
amended